schema.cc

```diff
@@ -656,6 +658,145 @@ std::ostream& operator<<(std::ostream& os, const schema& s) {
     return os;
 }
 
+std::ostream& map_as_cql_param(std::ostream& os, const std::map<sstring, sstring>& map, bool first = true) {
+    for (auto i: map) {
+        if (first) {
+            first = false;
+        } else {
+            os << ",";
+        }
+        os << "'" << i.first << "': '" << i.second << "'";
+    }
+
+    return os;
+}
+
+std::ostream& column_definition_as_cql_key(std::ostream& os, const column_definition & cd) {
+    os << cd.name_as_cql_string();
+    os << " " << cd.type->cql3_type_name();
+
+    if (cd.kind == column_kind::static_column) {
+        os << " STATIC";
+    }
+    return os;
+}
+
+static sstring remove_index_suffix(const sstring& name) {
+    return name.substr(0, name.size() - 6); // remove the _index suffix from an index name;
+}
+
+static bool is_secondery_index(const utils::UUID& id, const schema& s) {
```

Comment:
there's a typo here :) is_second**a**ry_index

Also, the commit message  has `schmea` instead of `schema`

Reply:
@psarna thanks, I've forced push V3 with a fix